backend.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  backend "s3" {
+    encrypt        = true
+    bucket         = "cisa-cool-terraform-state"
```

Comment:
I forget how this works.   Is this the actual bucket name?  Any issues if it is?

Reply:
It is the actual bucket name.  I have no issue with that.